CHANGELOG.md

```diff
@@ -1,233 +1,42 @@
-# 3.16.0 Tasty Taiyaki (2020-08-10)
+# 3.16.1 Tasty Taiyaki (2020-09-28)
```

Comment:
do we usually replace the entire changelog?

Reply:
Ah no, I'm guessing we shouldn't.  Will fix.

Reply:
Fixed!